packages/dht/src/identifiers.ts

```diff
@@ -8,22 +8,22 @@ export const KADEMLIA_ID_LENGTH_IN_BYTES = 20
 export type DhtAddress = BrandedString<'DhtAddress'>
 export type DhtAddressRaw = Uint8Array
 
-export const getDhtAddressFromRaw = (raw: DhtAddressRaw): DhtAddress => {
+export const toDhtAddress = (raw: DhtAddressRaw): DhtAddress => {
     return binaryToHex(raw) as unknown as DhtAddress
 }
 
-export const getRawFromDhtAddress = (address: DhtAddress): DhtAddressRaw => {
+export const toDhtAddressRaw = (address: DhtAddress): DhtAddressRaw => {
     return hexToBinary(address) as unknown as DhtAddressRaw
 }
 
-export const getNodeIdFromPeerDescriptor = (peerDescriptor: PeerDescriptor): DhtAddress => {
-    return getDhtAddressFromRaw(peerDescriptor.nodeId)
+export const toNodeId = (peerDescriptor: PeerDescriptor): DhtAddress => {
```

Comment:
feels like the conversion naming convention ("to") doesn't really fit here since the nodeId cannot be converted back to a PeerDescriptor. It feels more like a getter.